lib/usd/ui/layerEditor/layerTreeItem.cpp

```diff
@@ -140,37 +140,15 @@ void LayerTreeItem::populateChildren(RecursionDetector* recursionDetector)
     for (auto const path : subPaths) {
         std::string actualPath = SdfComputeAssetPathRelativeToLayer(_layer, path);
         auto        subLayer = SdfLayer::FindOrOpen(actualPath);
-        if (subLayer) {
```

Comment:
The difference in the if/else was merely to print error message in stdout, which are not useful and we already show error icon in the UI. Simplified the code.